plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/KuduQueryRunnerFactory.java

```diff
@@ -83,6 +83,12 @@ public static QueryRunner createKuduQueryRunnerTpch(TestingKuduServer kuduServer
         return createKuduQueryRunnerTpch(kuduServer, kuduSchemaEmulationPrefix, ImmutableMap.of(), ImmutableMap.of(), tables);
     }
 
+    public static QueryRunner createKuduQueryRunnerTpch(HostAndPort kuduServerMasterAddress, Optional<String> kuduSchemaEmulationPrefix, Iterable<TpchTable<?>> tables)
```

Comment:
This method is unused.

Reply:
This change is not required. I see that Kudu server is upgraded to 1.17. tests are running fine on Mac with this version.

Reply:
Yeah, i've fixed it on local